modules/k8s-nmstate-uninstall-operator.adoc

```diff
@@ -47,6 +48,23 @@ kubernetes-nmstate-operator.v4.18.0   Kubernetes NMState Operator   4.18.0
 $ oc delete --namespace openshift-nmstate clusterserviceversion kubernetes-nmstate-operator.v4.18.0
 ----
 
+. Before you delete the `nmstate` CR, remove the `nmstate-console-plugin` plugin resource from the `nmstate` CR so that `nmstate-console-plugin` does not persist.
++
+.. Create an environment variable that uses the `jq` CLI to find the location of `nmstate-console-plugin` among the list of available plugins. The following example assigns a command to an environment variable named `INDEX`:
++
+[source,terminal]
+----
+INDEX=$(oc get console.operator.openshift.io cluster -o json | jq -r '.spec.plugins | to_entries[] | select(.value == "nmstate-console-plugin") | .key')
+----
++
+.. Remove the `nmstate-console-plugin` resource from the `nmstate` CR by entering the following command in the `oc` CLI:
++
+[source,terminal]
+----
+$ oc patch console.operator.openshift.io cluster --type=json -p "[{\"op\": \"remove\", \"path\": \"/spec/plugins/$INDEX\"}]" <1>
+----
+<1> Replace `INDEX` with the name of your environment variable.
```

Comment:
no need for this comment. if you want to emphasize the name doesn't matter:
```suggestion
<1> `INDEX` is an auxiliary variable used in the second command. It can be named differently.
```

Reply:
:heavy_check_mark: